### PR TITLE
Tweaks around build tasks

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: packages/popmotion-pose/package.json [hunk 1]
```diff
@@ -47,6 +47,7 @@
     "ts-jest": "^22.4.5"
   },
   "dependencies": {
+    "hey-listen": "^1.0.5",
```
Comment (Andarist) [inline packages/popmotion-pose/package.json]: this was used but missing in the deps

File: packages/popmotion-pose/rollup.config.js [hunk 2]
```diff
@@ -5,13 +5,25 @@ import replace from 'rollup-plugin-replace';
 import commonjs from 'rollup-plugin-commonjs';
 import pkg from './package.json';
 
+const makeExternalPredicate = externalArr => {
```
Comment (Andarist) [inline packages/popmotion-pose/rollup.config.js]: this prevents rollup warnings around unresolved externals

File: packages/pose-core/package.json [hunk 3]
```diff
@@ -38,10 +38,7 @@
     "jest-cli": "^23.1.0",
     "prettier": "1.11.1",
     "rollup": "^0.59.1",
-    "rollup-plugin-node-resolve": "^3.3.0",
```
Comment (Andarist) [inline packages/pose-core/package.json]: those were unused in this package

File: packages/pose-core/rollup.config.js [hunk 4]
```diff
@@ -13,12 +10,12 @@ const makeExternalPredicate = externalArr => {
...
     return () => false;
   }
-  const pattern = new RegExp(`^(${externalArr.join("|")})($|/)`);
+  const pattern = new RegExp(`^(${externalArr.join('|')})($|/)`);
   return id => pattern.test(id);
 };
 
-const deps = Object.keys(pkg.dependencies || {})
-const peerDeps = Object.keys(pkg.peerDependencies || {})
+const deps = Object.keys(pkg.dependencies || {});
```
Comment (Andarist) [inline packages/pose-core/rollup.config.js]: linting issues

File: tsconfig.json [hunk 5]
```diff
@@ -11,6 +11,7 @@
     "watch": false,
     "baseUrl": "./packages",
     "paths": {
+      "@popmotion/easing": ["./easing/lib"],
```
Comment (Andarist) [inline tsconfig.json]: this was missing and preventing popmotion builds (which depends on this package)

**Comment by InventingWithMonster** [comment]
Excellent thanks for this!

**Pull request merged**